install/resources/mybb_theme.xml

```diff
@@ -12661,13 +12661,13 @@ if(use_xmlhttprequest == "1")
 {$footer}
 </body>
 </html>]]></template>
-		<template name="usercp_forumsubscriptions_forum" version="1816"><![CDATA[<tr>
-<td class="trow1" align="center" valign="top" width="1"><span class="forum_status forum_{$lightbulb['folder']}" title="{$lightbulb['altonoff']}"></span></td>
-<td class="trow1" valign="top">
+		<template name="usercp_forumsubscriptions_forum" version="1821"><![CDATA[<tr>
```

Comment:
Can you change this to `1820` please so that we can include this in 1.8.20?

Reply:
Done :)